rpcserver.go

```diff
@@ -491,26 +491,30 @@ func (r *rpcServer) MintAsset(ctx context.Context,
 	}
 }
 
-// checkFeeRateSanity ensures that the provided fee rate is above the same
-// minimum fee used as a floor in the fee estimator.
+// checkFeeRateSanity ensures that the provided fee rate, in sat/vB, is above
+// the same minimum fee used as a floor in the fee estimator.
 func checkFeeRateSanity(rpcFeeRate uint32) (*chainfee.SatPerKWeight, error) {
-	var feeRate *chainfee.SatPerKWeight
-	switch {
+	feeFloor := chainfee.FeePerKwFloor.FeePerKVByte()
```

Comment:
I think we should calculate in sat/kw only in here, so we should rather convert the manual fee rate below to sat/kw.

Reply:
Or rather, the manual fee rate should be in sat/kw on the RPC, see comment below.